### PR TITLE
Replace Nightly URLs with Current URLs

### PR DESCRIPTION
**Pull request by amayer5125**

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -7,7 +7,7 @@ Changelog
     file is managed by towncrier. You *may* edit previous change logs to
     fix problems like typo corrections or such.
     To add a new change log entry, please see
-    https://docs.pulpproject.org/en/3.0/nightly/contributing/git.html#changelog-update
+    https://docs.pulpproject.org/en/3.0/contributing/git.html#changelog-update
```
Comment (goosemania) [inline CHANGES.rst:10]: The current ones do not contain `en/3.0/`. E.g. https://docs.pulpproject.org/contributing/git.html#changelog-update

**Review by goosemania — CHANGES_REQUESTED** [review]
Thanks for improving the docs!

Could you update the links according to my comment?

File: CHANGES.rst (hunk 1 above)
Comment (goosemania) [reply CHANGES.rst:10]: Similar problem with the other links you changed.

Comment (amayer5125) [reply CHANGES.rst]: All links updated as recommended.

**Review by goosemania — APPROVED** [review]
Thanks!

**Pull request merged**